common/remote/sockfile.cpp

```diff
@@ -4365,12 +4431,71 @@ class CRemoteFileServer : public CInterface, implements IRemoteFileServer, imple
             listenep.getIpText(ips);
             acceptsock.setown(ISocket::create_ip(listenep.port,ips.str()));
         }
+        if (useSSL) {
+            querySecuritySettings(&useSSL, &configepPort, &clientCertificate.certificate, &clientCertificate.privateKey);
```

Comment:
Is this right? if (useSSL), then querySecuritySettings sets &useSSL ... ?

See other comment, if you had the settings set once in contructor of clientCertificate, then I think you avoid the need to recall.